docker-compose.yml

```diff
@@ -1,27 +1,24 @@
-version: "2.2"
+version: '3.8'
 
 services:
-  mysql:
-    image: mysql:8.2
+  # Application service
+  v5:
+    build: ./
+    environment:
+      - SPRING_PROFILES_ACTIVE=mysql
+      - MYSQL_USER=root
+      - MYSQL_PASS=petclinic
     ports:
-      - "3306:3306"
+      - "8080:8080"  # Expose application port
+    depends_on:
+      - mydb  # Ensure database service is started first
+
+  # Database service
+  mydb:
+    image: mysql:latest
     environment:
-      - MYSQL_ROOT_PASSWORD=
-      - MYSQL_ALLOW_EMPTY_PASSWORD=true
-      - MYSQL_USER=petclinic
-      - MYSQL_PASSWORD=petclinic
       - MYSQL_DATABASE=petclinic
-    volumes:
-      - "./conf.d:/etc/mysql/conf.d:ro"
-    profiles:
-      - mysql
-  postgres:
-    image: postgres:16.1
+      - MYSQL_USER=petclinic
+      - MYSQL_ROOT_PASSWORD=petclinic 
```

Comment:
This would work by coincidence, however I see some mismatch here.
You are passing different set of variables with different values to different containers.
Try to think about variables that are passed to containers and we'll discuss this verbally tomorrow.